Frontend/src/Components/Graph/GraphSelectModal.js

```diff
@@ -85,37 +91,45 @@ function GraphSelectModal(props) {
   const [historyLength, setHistoryLength] = useState(initialHistoryLength);
 
   return (
-    <Modal isOpen={isOpen} onClose={onClose} size="xl">
+    <Modal isOpen={isOpen} onClose={onClose} size="3xl">
       <ModalOverlay />
       <ModalContent>
         <ModalHeader>Select Datasets</ModalHeader>
         <ModalCloseButton />
-
         <ModalBody>
           <VStack align="stretch">
-            {GraphData.categories.map((category) => (
+            <hr />
```

Comment:
```suggestion
            <hr/>
```
nit: Up to you, just personal preference

Reply:
fixed